src/components/PostInfo/PostInfo.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+
+import { Post } from '../../types/Post';
+import { UserInfo } from '../UserInfo/UserInfo';
+import { CommentList } from '../CommentList/CommentList';
+
+type Props = {
+  post: Post;
+};
+
+export const PostInfo: React.FC<Props> = ({ post }) => {
+  return (
+    <>
+      <h2 className="postList__title">
+        {post.title}
+      </h2>
+
+      <div className="postList__description">
+        {post.body}
+      </div>
+
+      <div className="postList__userInfo">
+        {post.user && <UserInfo user={post.user} />}
+      </div>
```

Comment:
Maybe better don't render empty div if the post doesn't have a writer? And should comments fix too